Source/ToolbarAction.m

```diff
@@ -0,0 +1,35 @@
+//
+//  ToolbarAction.m
+//  Pods
+//
+//  Created by Matt Bush on 8/29/16.
+//
+//
+
+#import "ToolbarAction.h"
+
+@implementation ToolbarAction
+
+- (void)setBackgroundImage:(NSString *)imageName
+{
+    [self.imageView setContentMode: UIViewContentModeScaleAspectFill];
```

Comment:
Yeah, it should always fit perfectly based on constraints - more so just a double check to make sure that the provided image is both taking up as much space in the container as possible and respecting its aspect ratio.